File: java-source/src/test/java/net/corda/accord/contract/PromissoryNoteTransferTests.java
package net.corda.accord.contract;

import net.corda.core.contracts.CommandData;
import net.corda.core.contracts.TypeOnlyCommandData;
import net.corda.testing.node.MockServices;

import java.util.Arrays;

/**
 * TODO: Modify tests to adapt from IOU to promissory note.
 */

public class PromissoryNoteTransferTests {

    public interface Commands extends CommandData {
        class DummyCommand extends TypeOnlyCommandData implements Commands{}
    }

    static private final MockServices ledgerServices = new MockServices(Arrays.asList("net.corda.accord"));

}

File: kotlin-source/src/test/kotlin/net/corda/accord/contract/PromissoryNoteTransferTests.kt
package net.corda.accord.contract

import net.corda.core.contracts.CommandData
import net.corda.core.contracts.ContractState
import net.corda.core.identity.AbstractParty
import net.corda.testing.node.MockServices


class PromissoryNoteTransferTests {
    // A pre-made dummy state we may need for some of the tests.
    class DummyState : ContractState {
        override val participants: List<AbstractParty> get() = listOf()
    }
    // A dummy command.
    class DummyCommand : CommandData
    var ledgerServices = MockServices(listOf("net.corda.accord"))

}